src/pages/SearchHelpPage.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+<div class="grid-container full">
+  <header-section showNavSearch="true"></header-section>
+  <div class="page_body">
+    <h1>CC Search Advanced Search User Guide</h1>
```

Comment:
General feedback about the text: I think the descriptions of each term are a little too long, I think it would be easier to read quickly if we made them shorter like https://support.google.com/websearch/answer/2466433?visit_id=636963999175252541-459155138&p=adv_operators&hl=en&rd=1